web/forms.py

```
@@ -213,6 +218,20 @@ def __init__(self, *args, **kwargs):
 
+            # Special handling for username field
+            if "username" in self.data:
+                if "username" in self.fields:
+                    self.fields["username"].widget.attrs["value"] = self.data["username"]
+                else:
+                    # Re-add the username field since allauth removed it
+                    self.fields["username"] = forms.CharField(
+                        max_length=30,
+                        required=True,
+                        widget=TailwindInput(
+                            attrs={"placeholder": "Choose a username", "value": self.data["username"]}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)